### PR TITLE
feat(domains)

### PR DESCRIPTION
**Pull request by Geo25rey**

### CODE REVIEW

File: src/commands/domains/add.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+              'Please add the following DNS record to your domain,',
+              id + ',',
+              'to verify your ownership',
+            )
+            this.log()
+            this.log('Type: TXT')
+            this.log('Name:', record_name)
+            this.log('Value:', record_value)
+            this.log()
+            this.log('Type: MX')
```
Comment (vaultec81) [inline src/commands/domains/add.ts:63]: What about DKIM and DMARC?

File: src/commands/domains/delete.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      this.log('Could not connect to API. Check your internet connection')
+      this.exit(1)
+    }
+
+    return this.delete(client, flags)
+  }
+
+  async delete(client: typeof api, { domain }: FlagsType): Promise<void> {
+    if (!domain) {
+      this.log('Please provide a domain to be setup')
```
Comment (vaultec81) [inline src/commands/domains/delete.ts:43]: Please provide a domain to **delete**

File: src/commands/domains/add.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+export default class DomainsAdd extends Command {
+  static description = 'add a domain'
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    domain: flags.string({
+      char: 'd',
+      description: 'the domain to add to your account',
```
Comment (vaultec81) [inline src/commands/domains/add.ts:22]: Capitalize the. Double check all areas which may need capitalization.

File: src/commands/domains/delete.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import { Command, flags } from '@oclif/command'
```
Comment (vaultec81) [inline src/commands/domains/delete.ts:1]: I would assume the lint-test would need to get fixed as well.

Comment (Geo25rey) [reply src/commands/domains/delete.ts:1]: Later

Comment (vaultec81) [reply src/commands/domains/delete.ts:1]: Yea non essential at the moment

File: src/commands/domains/add.ts (hunk 1 above)
Comment (Geo25rey) [reply src/commands/domains/add.ts:63]: Not sure, shouldn't that be provided by the api? Wait for me to finish current work on api before potential changes.

File: src/commands/domains/add.ts (hunk 3 above)
Comment (Geo25rey) [reply src/commands/domains/add.ts:22]: Sticking with the rest of the help command description formatting

Comment (vaultec81) [reply src/commands/domains/add.ts:22]: Oh I was unaware that is formatted that way.

File: src/commands/domains/add.ts (hunk 1 above)
Comment (vaultec81) [reply src/commands/domains/add.ts:63]: DKIM and DMARC is assumed to be static. If they do change then we would have to notify the user of such changes a whole separate process. I think for now this isn't important... On another note we can provide a link to the docs https://docs.mailscript.com/cli/domains/

**Pull request merged**